Project Management Process.md

```diff
@@ -0,0 +1,168 @@
+
+### Milestones
+A milestone is a group of project's goals. Each project has a set of milestones. A milestone is completed when all goals associated with it have been completed. Milestones are tracked in the project roadmap by the project lead.
+
+A milestone has the following information:
+- Name
+  - TODO: describe what is a good name, give an example
+- Due date
+- List of goals attached to it
+- Progress indicator
+
+The Project Lead keeps current milestones up to date on weekly-basis. New milestones should be generated and old milestones updated before the quarterly planning meeting. Prepare new milestones in good time before quarterly planning.
+
+TODO: a clear example
+
+### Project Roadmap
+The milestones are tracked in project's roadmap. The project lead owns the roadmap and is responsible for generating, updating and tracking the milestones. To make sure the roadmap is up to date, the project lead should go through the current milestones on weekly basis.
+
+The Roadmap Document contains the following information:
+- List of current milestones
+  - Ordered by expected due date, next upcoming milestone at the top
+- Current status of the project, ie. milestone progress
+- List of previous, recent milestones (< 6 months)
+- List of links to very old milestones (> 6 months)
+
+The roadmap can be a high-level overview of milestones and project's progress, so details suchs as list of goals per milestone can be omitted. However, if omitted, the roadmap must contain links to the detailed break down.
+
+Examples:
+- [js-ipfs roadmap](https://github.com/ipfs/js-ipfs/blob/master/ROADMAP.md#ipfs-javascript-implementation-roadmap)
+- [Orbit roadmap](https://github.com/haadcode/orbit/blob/master/ROADMAP.md#orbit---roadmap)
+
+### Organization Roadmap
+An overview of all projects across the organization is tracked in organization's roadmap. This is similar to a project roadmap, but collects all milestones from all projects into one place.
+
+Organization roadmap contains the following information:
+- List of all projects
+- List of all current milestones and their progress in each project
+- List of links to old milestones
+
+The organization roadmap is owned by project management and project managers are responsible for generating and keeping the roadmap up to date. The roadmap should be updated in the quarterly planning meeting as old milestones are reviewed and the project leads commit to new milestones.
+
+TODO: a clear example
+
+## Flow
+
+### Quarterly Planning
+TODO
+
+What:
+- Identify project’s / company’s needs and main efforts.
```

Comment:
Please just remove reference to "company". say "Organization", as "The IPFS Project" or "IPFS Team/Community" is an organization. even though it does not yet have legal standing. (and it will eventually :) )